xsgen/openmc_origen.py

```diff
@@ -605,9 +611,15 @@ def _generate_xs(self, e_g, phi_g):
         for nuc in nucs:
             for rx in rxs:
                 xs = xscache[nuc, rx, temp]
+                if(len(xs) < 100):
```

Comment:
So, for some reason some of the XS arrays are not 100 energy groups. None of the ones like this are useful to us right now so I just put in this continue.